### ISSUE TITLE
Where did the Authorize attribute for HC.AspNetCore.Authorization go?

### ISSUE THREAD

**Issue by jkears** [ChilliCream/graphql-platform#5852]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Product

Hot Chocolate

### Describe the bug

Just attempted to switch from 12.8.x to 13.0.2 and now unable to compile because no longer seeing the Authorization Attribute.

![image](https://user-images.githubusercontent.com/1290937/219395489-9fb0d445-c269-48c6-931f-944c251132db.png)

What is the work-around?

### Steps to reproduce

1. Add a package ref. to HotChocolate.AspNetCore.Authorization.Authorize
2. Add Authorize attribute (no longer exists).

### Relevant log output

```shell
NA
```

### Additional Context?

NA

### Version

13.0.2

**Comment by KHaertner** [comment]
You can try `HotChocolate.Authorization.Authorize` instead of `HotChocolate.AspNetCore.Authorization.Authorize`

**Comment by jkears** [comment]
Thank you @KHaertner that seemed to get us through that concern but my goodness this 13.x release has so many breaking changes.  We are getting major startup issues.  Not sure where to turn.  Is there a document somewhere that describes all of the breaking changes as this is a crap experience.  We have never had so many breaking changes when migrating on a major release.

**Comment by KHaertner** [comment]
The best I found for migration to v13 is [this](https://chillicream.com/docs/hotchocolate/v13/migrating/migrate-from-12-to-13)

**Comment by PascalSenn** [comment]
@jkears  this is weird. I mean, the Authorize attribute namespaces is expected to be breaking. But the startup issues you experience are not expected. 
Do you have static methods on your models?
Can you share  `ApplicationPage`?
![image](https://user-images.githubusercontent.com/14233220/219596569-a7a9f96a-a8f3-4285-8dc8-97e040f5e254.png)

**Comment by PascalSenn** [comment]
Closed this issue as you have already opened one for the other thing

**Comment by michaelstaib** [comment]
@jkears be aware that authorization completely changed in behavior ... we have some content coming up on this. We took in all the user feedback over the last years and applied for auth. Auth is the biggest change. Apart form this I would say the upgrade experience should be smooth.

We outlined the breaking changes here:
https://chillicream.com/docs/hotchocolate/v13/migrating/migrate-from-12-to-13
